README.md

```diff
@@ -48,6 +49,69 @@ ManUpStatus result = await service.validate();
 - `unsupported`: The app is an unsupported version and should not run
 - `disabled`: The app has been marked as disabled and should not run
 
+### Using the Service with Delegate
+Implement `ManupDelegate` or use `ManupDelegateMixin` mixin which has default implementation.
+  `void manUpConfigUpdateStarting()` : will be called before starting to validate.
+  `void manupStatusChanged(ManUpStatus status)` : will be called every time status changes.
+  `void manUpUpdateAvailable()` : will be called when ManUpStatus changes to supported
+  `void manUpUpdateRequired()` : will be called when ManUpStatus changes to unsupported
+  `void manUpMaintenanceMode()`: will be called when ManUpStatus changes to disabled
```

Comment:
The formatting in this section needs to be fixed 

https://github.com/NextFaze/flutter_manup/blob/f3a58664c009ef2a089b2968a6ab3b519b3aef59/README.md#using-the-service-with-delegate